rewrite/pyproject.toml

```diff
@@ -1,27 +1,38 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
 name = "openrewrite"
-version = "0.0.0"  # This will be replaced by the GitHub Actions workflow
 description = "OpenRewrite is a mass automated refactoring engine."
-authors = ["Moderne Inc. <support@moderne.io>"]
-license = "Apache-2.0"
-packages = [
-    { include = "rewrite" }
+version = "0.0.0" # Updated dynamically during release
+authors = [{ name = "Moderne Inc.", email = "support@moderne.io" }]
+license = { text = "Moderne, Inc. Commercial License" }
+dependencies = [
+    "pytest>=8.3.4",
+    "more-itertools>=10.4.0",
+    "openrewrite-remote",
+    "pip>=24.3.1",
+    "pypi-simple>=1.6.1",
```

Comment:
probably dont need these; I believe their use is specific to remoting project

Reply:
Sounds good. I indeed just copied this.